functions/package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "playMedia",
+  "name": "audio-interface-of-internet-archive",
```

Comment:
I don't get why we need to change `functions/package-lock.json` in this branch, can't we just skip those changes?

Reply:
Actually, I used master to after reset to avoid conflict. Will it make any impact?